corehq/form_processor/models/util.py

```diff
@@ -14,3 +15,17 @@ def key(obj):
 
     index_map = {id_: index for index, id_ in enumerate(id_list)}
     object_list.sort(key=key)
+
+
+def attach_prefetch_models(objects_by_id, prefetched_models, link_field_name, cached_attrib_name):
```

Comment:
should `corehq.form_processor.backends.sql.dbaccessors._attach_prefetch_models` be deleted?

Reply:
Yes, that will happen in a future PR as part of the case accessors refactoring.